recipes/mujoco/build_cxx.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+mkdir build
+cd build
+
+cmake ${CMAKE_ARGS} -GNinja .. \
+      -DCMAKE_BUILD_TYPE=Release \
+      -DBUILD_SHARED_LIBS:BOOL=ON \
+      -DMUJOCO_BUILD_TESTS:BOOL=ON \
+      -DMUJOCO_BUILD_EXAMPLES:BOOL=OFF \
+      -DMUJOCO_ENABLE_AVX:BOOL=OFF \
+      -DMUJOCO_ENABLE_AVX_INTRINSICS:BOOL=OFF \
```

Comment:
Please consider adding `-DCMAKE_INTERPRODEDURAL_OPTIMIZATION:BOOL=ON` as well.

Reply:
typo: should be `-DCMAKE_INTERPROCEDURAL_OPTIMIZATION`

Reply:
Thanks, done in https://github.com/traversaro/staged-recipes/commit/7575534d1b7cd3542967c0d21e615c93c8221222 and https://github.com/traversaro/staged-recipes/commit/a5957f4325fd0a6ff97175e3f67837e4aea40f04 .